docs/docsite/rst/network/user_guide/platform_nxos.rst

```diff
@@ -0,0 +1,125 @@
+.. _nxos_platform_options:
+
+***************************************
+NXOS Platform Options
+***************************************
+
+Cisco NXOS supports multiple connections. This page offers details on how each connection works in Ansible 2.5 and how to use it.
+
+.. contents:: Topics
+
+Connections Available
+================================================================================
+
++---------------------------+-----------------------------------------------+-----------------------------------------+
+|..                         | CLI                                           | nxAPI                                   |
```

Comment:
References to `nxAPI` would be better to use `NX-API` instead.

Reply:
Thanks for the feedback. Incorporated in 9f44f34c57d5db1547e6018784a5feb297685061.